app/views/inventories/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <h1>Inventory Details</h1>
+
+  <div class="inventory_details">
+    <p><strong>Name:</strong> <%= @inventory.name %></p>
+
+    <h2>Added Foods:</h2>
+    <ul>
+      <% @inventory.inventory_foods.each do |inventory_food| %>
+        <li>
+          <% if inventory_food.food.present? %>
+            <%= inventory_food.food.name %> - Quantity: <%= inventory_food.quantity %>
+            <%= button_to 'Remove', user_inventory_inventory_food_path(@user, @inventory, inventory_food), method: :delete, data: { confirm: 'Are you sure?' }, class: 'button' %>
+          <% else %>
+            No food information available
+          <% end %>
+        </li>
+      <% end %>
+    </ul>
+  </div>
+
+  <%= link_to 'Back to Inventories List', user_inventories_path(@user) %>
+  <h2>Add Food to Inventory</h2>
+  <%= link_to 'Add Food', new_user_inventory_inventory_food_path(@user, @inventory), class: 'button' %>
+</div>
```

Comment:
- [ ] Your design should match with the `wireframe` as I mentioned earlier, it seems like you missed the basic styling to achieve the `wireframe` design. Kindly make your `detail` page according to the design provided in the requirement.

## Design should look like

![image](https://github.com/DevKaranJ/recipes/assets/66476329/28befb18-3136-4b2a-b429-19da75e88708)

## Your Design
![image](https://github.com/DevKaranJ/recipes/assets/66476329/309587ed-eff9-49b1-a33a-d31f731a8e85)